packages/rrweb/src/replay/index.ts

```diff
@@ -614,13 +631,6 @@ export class Replayer {
     this.wrapper.classList.add('replayer-wrapper');
     this.config.root.appendChild(this.wrapper);
 
-    if (this.config.mouseTail !== false) {
```

Comment:
deleted this bit because this only gets called in the constructor and at that point we don't have any mousetails yet